src/components/settings/SettingsAccessTab/index.jsx

```diff
@@ -91,47 +91,43 @@ const SettingsAccessTab = ({
           )}
         </div>
       )}
-      {hasConfiguredSubsidyType && (
```

Comment:
The changes in this file are really mostly indentation changes. I wanted to remove 
```
{hasConfiguredSubsidyType && (	
        <>
```
from the code, and then all the following lines were outdented (at least, that is what I am aiming for!)